mitodl/mit-open
# content file filters

# What are the relevant tickets?
closes https://github.com/mitodl/mit-open/issues/284

# Description (What does it do?)
This pr updates the filter names for the content file api, specifically
run  -> run_id
run__learning_resource	-> learning_resource_id
run__learning_resource__offered_by__name -> 	offered_by
run__learning_resource__platform -> platform	 
run__learning_resource__readable_id	-> learning_resource_readable_id
run__run_id	-> run_readable_id

It also adds content_feature_type as a filter

# How can this be tested?
Go to http://localhost:8063/api/v1/contentfiles/ 
Click the filters button and try out the filters
Go to http://localhost:8063/api/v1/schema/swagger-ui/#/contentfiles/contentfiles_list 
Make sure the documentation makes sense
